### PR TITLE
beta9 relnotes

### PR DESCRIPTION
**Pull request by tardyp**
for your kind review before creating the release.

Please have a look for my spaling mistooks

### CODE REVIEW

**Comment by tardyp** [comment]
https://circle-artifacts.com/gh/buildbot/buildbot/640/artifacts/0/home/ubuntu/buildbot/master/docs/_build/html/relnotes/0.9.0b9.html

**Comment by Riziero** [comment]
Do we guys drop the previous beta version from pip? I had a docker file
installing b07 and now no matter what I get

'Starting BuildMaster -- buildbot.version: 0.9.0b8'

Tried to clean up all the images on my dev env still getting the above
after rebuilding them.

File: master/docs/relnotes/0.9.0b9.rst [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+* ``buildbot stop`` now waits for complete buildmaster stop by default.
+
+* New ``--no-wait`` argument for ``buildbot stop`` which allows not to wait for complete master shutdown.
+
+* Builder page is now sorted by builder name
+
+* LogViewer page now supports ansi color codes, and is displayed white on black.
+
+* This is the last version of buildbot nine which supports python 2.6 for the master.
```
Comment (tomprince) [inline master/docs/relnotes/0.9.0b9.rst]: This seems like it should be in the deprecations sections?

File: master/docs/relnotes/0.9.0b9.rst [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+* Web server does not provide /png and /redirect anymore (:bug:`3357`).
+  This functionality is used to implement build status images.
+  This should be easy to implement if you need it.
+  One could port the old image generation code, or implement a redirection to http://shields.io/.
+
+* Support of worker-side ``usePTY`` was removed from ``buildbot-worker``.
+  ``usePTY`` argument was removed from ``WorkerForBuilder`` and ``Worker`` classes.
+
+* html is not permitted anymore in 'label' attributes of forcescheduler parameters.
```
Comment (tomprince) [inline master/docs/relnotes/0.9.0b9.rst]: s/is not permitted anymore/is no longer permitted/

File: master/docs/relnotes/0.9.0b9.rst (hunk 1 above)
Comment (tardyp) [inline master/docs/relnotes/0.9.0b9.rst]: well this one still features python 2.6, as we did not yet merged your patches. Maybe I should put it in an additional Note section.

**Comment by tardyp** [comment]
@Riziero for beta releases, pypi automatically hide older beta releases. if you really care about b7, you can point to https://pypi.python.org/packages/06/f3/656e5f676b6698cbf58a9da5887c0c1369602da589fbbad62607b00cf1fa/buildbot-0.9.0b7.tar.gz#md5=dba0296f3a2296b71b729b3abd8cfdaa

File: master/docs/relnotes/0.9.0b9.rst (hunk 1 above)
Comment (tomprince) [inline master/docs/relnotes/0.9.0b9.rst]: Well, so it is a deprecation but not yet a removal.

**Comment by tomprince** [comment]
You can also do 'pip install buildbot==0.9.0b7` to get a specific version.

**Comment by tardyp** [comment]
fixed spelling

**Comment by tardyp** [comment]
Note that I am holding the release until we find out more about http://trac.buildbot.net/ticket/3546

**Pull request merged**